modules/tinymce/src/plugins/fullscreen/main/ts/core/Actions.ts

```diff
@@ -112,14 +112,21 @@ const toggleFullscreen = (editor: Editor, fullscreenState: Cell<ScrollInfo | nul
   const iframe = editor.iframeElement;
   const iframeStyle = iframe.style;
 
+  const handleClasses = (handler: (elm: string | Node | Node[], cls: string) => void) => {
+    handler(body, 'tox-fullscreen');
+    handler(documentElement, 'tox-fullscreen');
+    handler(editorContainer, 'tox-fullscreen');
+    if (SugarShadowDom.isOpenShadowHost(fullscreenRoot)) {
```

Comment:
This isn't the correct function to use, as this needs to work with a closed shadow root as well. You probably need to do something like this instead:
```
SugarShadowDom.getShadowRoot(SugarElement.fromDom(editorContainer)).map((root) => {
  const host = SugarShadowDom.getShadowHost(root);
  handler(host.dom, 'tox-fullscreen tox-shadowhost');
})
```

Reply:
Thanks for the suggestion. This should be fixed now.